index.html

```diff
@@ -178,8 +181,8 @@ <h2>Scope</h2>
         <p>The Working Group may also publish new Working Group Notes that may be necessary to clarify, help the deployment and usage, or propose new features to the Group’s main recommendation-track deliverable.</p>
         <p>
 The Working Group may also publish new Working Group Notes, Editors Drafts, or
-Working Drafts of DID Method Specifications. Any documents intended for
-Recommendation will be limited to the Working Draft maturity level.
+Working Drafts for DID Resolution or of DID Method Specifications. Any documents
```

Comment:
```suggestion
Working Drafts for DID Resolution (incl. DID URL Dereferencing) or of DID Method Specifications. Any documents
```

Reply:
addressed in [409f1c6](https://github.com/w3c/did-wg-charter/pull/25/commits/409f1c61b8000805fe33cb67e8e8ee43ad8bf25a)